site/docs/samples/delete-sample.md

```diff
@@ -81,7 +81,13 @@ You should see the new created object:
 
 In this step you are performing the role of the data owner, registering his data in the data catalog and registering the credentials for accessing the data in the credential manager.
 
-Register the credentials required for accessing the dataset as a kubernetes secret. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+In this tutorial, we assume that OpenMetadata is used as the data catalog. Datasets can be registered either directly, through the OpenMetadata UI, or indirectly, through the data-catalog connector.
+
+If you use OpenMetadata directly through the UI, and use its discovery mechanism to add asset tables, you can determine an asset ID through the corresponding OpenMetadata table URL. If, for instance, the URL for the table is: `http://localhost:8585/table/openmetadata-s3.default.demo."PS_20174392719_1491204439457_log.csv"`, then the corresponding asset ID is `"openmetadata-s3.default.demo.\"PS_20174392719_1491204439457_log.csv\""`.
+
+If you registered the dataset directly through the OpenMetadata UI, you can skip to the next section.
+
```

Comment:
Same comments about UI and section heading as in the read scenario, or else link to one place where it is shown